src/web/views/Guidelines/Utilities/Utilities.vue

```diff
@@ -112,62 +100,60 @@ export default {
-        ["bg-high", "background-color: #ff3d1e;"],
-        ["color-primary", "color: #2f374a;"],
-        ["color-secondary", "color: #f9c72c;"],
-        ["color-silver", "color: #d8d8d8;"],
-        ["color-silver-light", "color: #f7f7f7;"],
-        ["color-silver-dark", "color: #bdbdbd;"],
-        ["color-granite-light", "color: #7a7a7a;"],
-        ["color-granite", "color: #606060;"],
-        ["color-white", "color: #ffffff;"],
-        ["color-black", "color: #000000;"],
-        ["color-success", "color: #00af50;"],
-        ["color-warning", "color: #ff9100;"],
-        ["color-high", "color: #ff3d1e;"],
+      columns: [
+        { id: "name", label: "Class name", width: "300px" },
+        { id: "value", label: "Output value" },
       ],
-      flexboxUtilities: [
-        ["Class name", "Output value"],
-        ["flex", "display: flex;"],
-        ["flex-row", "flex-direction: row;"],
-        ["flex-col", "flex-direction: column;"],
-        ["items-start", "align-items: flex-start;"],
-        ["items-center", "align-items: center;"],
-        ["items-end", "align-items: flex-end;"],
-        ["content-start", "align-content: flex-start;"],
-        ["content-center", "align-content: flex-center;"],
-        ["content-end", "align-content: flex-end;"],
-        ["content-between", "align-content: flex-between;"],
-        ["content-around", "align-content: flex-around;"],
-        ["justify-start", "justify-content: flex-start;"],
-        ["justify-center", "justify-content: flex-center;"],
-        ["justify-end", "justify-content: flex-end;"],
-        ["justify-between", "justify-content: flex-between;"],
-        ["justify-around", "justify-content: flex-around;"],
+      colorUtilities: [
+        ["bg-primary", "background-color: #2f374a"],
+        ["bg-secondary", "background-color: #f9c72c"],
+        ["bg-silver", "background-color: #d8d8d8"],
+        ["bg-silver-light", "background-color: #f7f7f7"],
+        ["bg-silver-dark", "background-color: #bdbdbd"],
+        ["bg-granite", "background-color: #606060"],
+        ["bg-granite-light", "background-color: #7a7a7a"],
+        ["bg-white", "background-color: #ffffff"],
+        ["bg-black", "background-color: #000000"],
+        ["bg-success", "background-color: #00af50"],
+        ["bg-warning", "background-color: #ff9100"],
+        ["bg-high", "background-color: #ff3d1e"],
+        ["color-primary", "color: #2f374a"],
+        ["color-secondary", "color: #f9c72c"],
+        ["color-silver", "color: #d8d8d8"],
+        ["color-silver-light", "color: #f7f7f7"],
+        ["color-silver-dark", "color: #bdbdbd"],
+        ["color-granite-light", "color: #7a7a7a"],
+        ["color-granite", "color: #606060"],
+        ["color-white", "color: #ffffff"],
+        ["color-black", "color: #000000"],
+        ["color-success", "color: #00af50"],
+        ["color-warning", "color: #ff9100"],
+        ["color-high", "color: #ff3d1e"],
```

Comment:
Following my previous comments, I think we can add color-neutral here (color: #205dbd")